website/docs/r/compute_router_nat.html.markdown

```diff
@@ -87,6 +87,7 @@ resource "google_compute_router_nat" "advanced-nat" {
   source_subnetwork_ip_ranges_to_nat = "LIST_OF_SUBNETWORKS"
   subnetwork {
     name = "${google_compute_subnetwork.subnetwork.self_link}"
```

Comment:
Do you mind bumping the `=` in this line up so it lines up with the argument below?

Reply:
@rileykarson : fixed it and squashed down to a single commit (8816d3a)

Reply:
Thanks! I'll upstream this change to Magic Modules so it hits `google-beta` too, and merge this PR.